test/integration/package_int_test.go

```diff
@@ -701,6 +701,86 @@ func (suite *PackageIntegrationTestSuite) TestProjectWithOfflineInstallerAndDock
 	cp.ExpectExitCode(0)
 }
 
+func (suite *PackageIntegrationTestSuite) TestImportWithCustomVersionRequirements() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("Skipping windows as builds can be slow and we only really need to test this one one platform")
+		return
+	}
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	// Because the ActiveState-CLI org has enterprise features enabled, we need to login
+	ts.LoginAsPersistentUser()
+
+	complexReqsData := `coverage!=3.5
+docopt>=0.6.1
+Mopidy-Dirble>=1.1,<2
+requests>=2.2,<2.31.0
+urllib3>=1.21.1,<=1.26.5
+`
+
+	cp := ts.Spawn("checkout", "ActiveState-CLI/Comparators-Import", ".")
+	cp.Expect("Skipping runtime setup")
+	cp.Expect("Checked out project")
+	cp.ExpectExitCode(0)
+
+	ts.PrepareFile(filepath.Join(cp.WorkDirectory(), reqsFileName), complexReqsData)
+
+	cp = ts.Spawn("import", reqsFileName)
+	cp.ExpectExitCode(0)
+
+	// Test version numbers once the following story is resolved:
+	// https://activestatef.atlassian.net/browse/DX-2272
+	cp = ts.Spawn("packages")
+	cp.Expect("Mopidy-Dirble")
+	cp.Expect("coverage")
+	cp.Expect("docopt")
+	cp.Expect("requests")
+	cp.Expect("urllib3")
```

Comment:
Since you can't use `state packages` and since verifying the version numbers is the point of this tests maybe instead we can use `state history`?

I don't see any point in adding the test as is because again it is not testing what it is set out to test.